db/migrate/20191023180308_subclass_ems_cluster.rb

```diff
@@ -20,7 +22,6 @@ def up
       ems_class_name = "ManageIQ::Providers::#{provider}::InfraManager"
       cluster_class_name = "#{ems_class_name}::Cluster"
       EmsCluster.in_my_region.
```

Comment:
since the `where()` clause has `:ext_management_systems`, you still need the `joins(:ext_management_systems)`